build.sbt

```diff
@@ -297,7 +296,7 @@ def nativeImageDockerSettings: Seq[Setting[_]] = dockerSettings ++ Seq(
   nativeImageDockerBuild := false,
   // If this is Some(…): run the native-image generation inside a Docker image
   // If this is None: run the native-image generation using a local GraalVM installation
-  graalVMVersion := Some(GraalVersion),
+  graalVMVersion := None, //Some(GraalVersion),
```

Comment:
@dwijnand This will run the local graal version, so if that's not 20.0.0 then you'll want to change this